controllers/articles.controllers.js

```diff
@@ -58,3 +59,15 @@ exports.postArticleComment = (request, response, next) => {
       next(err);
     });
 };
+
+exports.patchArticleVotes = (request, response, next) => {
+  const { article_id } = request.params;
+  const votes = request.body;
```

Comment:
I think there's a case for destructuring `inc_votes` here, rather than pointing to `request.body`, and dealing with the value directly